guides/doc-Configuring_DNS_DHCP_TFTP/docinfo.xml

```diff
@@ -0,0 +1,19 @@
+<title>{ConfiguringDNSDHCPTFTPDocTitle}</title>
+<productname>Red Hat Satellite</productname>
+<productnumber>{ProjectVersion}</productnumber>
+<subtitle>Configure DNS, DHCP, and TFTP services or disable them</subtitle>
```

Comment:
"or disable them" sounds weird because its counterpart is "enable", not "configure". 

However, this PR is just to create placeholders.